recipes/netcdf-cxx-legacy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "4.2" %}
+
+package:
+  name: netcdf-cxx-legacy
+  version: {{ version }}
+
+source:
+  url: https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-cxx-{{ version }}.tar.gz
+  sha256: 95ed6ab49a0ee001255eac4e44aacb5ca4ea96ba850c08337a3e4c9a0872ccd1
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - libnetcdf
+    - hdf5
+  run:
+    - libnetcdf
+    - hdf5
+  run_constrained:
+    - netcdf-cxx4 >=9999
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libnetcdf_c++.a
+    - test -f $PREFIX/lib/libnetcdf_c++.so  # [linux]
```

Comment:
There is an SOEXTNAME env var or something like that that can make this line a bit nicer. shrug on using it or not. I like the clarity here.

Reply:
It is `SHLIB_EXT` I think.

https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html

Reply:
Ah, great!  I vaguely knew there was a way but didn't find it myself searching around.  I"m mostly a python programmer these days...